index.html

```diff
@@ -1494,12 +1532,24 @@ <h2>Core Vocabulary Definitions</h2>
 </p>
 </section>
 <section><h3><code>VersionInfo</code></h3><p>Carries version information about the TD instance. If required, additional version information such as firmware and hardware version (term definitions outside of the TD namespace) can be extended here.</p><table class="def"><thead><tr><th><a>Vocabulary term</a></th><th>Description</th><th>Assignment</th><th>Type</th></tr></thead><tbody><tr class="rfc2119-table-assertion" id="td-vocab-instance--VersionInfo"><td><code>instance</code></td><td>Provides a version identicator of this TD instance.</td><td>mandatory</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a></td></tr></tbody></table></section>
-<section><h3><code>MultiLanguage</code></h3><p>Container to provide human-readable text in different languages using language tags described in [[BCP47]]. Each used language tags code MUST be decleared as vocabulary term member of this container (e.g., en, de, ja, zh-Hans, zh-Hant). See <a href="#titles-descriptions-serialization-json"></a> for example usages of this container in a Thing Description instance.</p>
+<section><h3><code>MultiLanguage</code></h3><p>Container to provide human-readable text in different languages using language tags described in [[BCP47]]. See <a href="#titles-descriptions-serialization-json"></a> for example usages of this container in a Thing Description instance.</p>
 <p>
+<span class="rfc2119-assertion" id="td-context-ns-multilanguage-language-tag">
+Each language tag code used MUST be declared as vocabulary term member of this container 
+(e.g., <code>en</code>, <code>de</code>, <code>ja</code>, <code>zh-Hans</code>, <code>zh-Hant</code>,
+<code>az-Latn</code>, <code>az-Arab</code>). </span>
+<span class="rfc2119-assertion" id="td-context-ns-multilanguage-text-direction-infer">
+The base text direction MUST be inferred from the language tag code.
```

Comment:
strike _code_

Reply:
This is a good candidate for Section 6.

Reply:
Code markup removed.   I also added Azeri as an example of where this is needed.  I am not sure this goes in Section 6, as it is not a serialization issue.   Text direction is how the text is displayed when it is rendered, i.e. how the data is to be interpreted.   Therefore you also need the text direction in the text's information model.

Reply:
I left this in the MultiLanguage section, and as discussed below, removed a redundant assertion in Section 6.